android/src/main/kotlin/com/rly_network/rly_network_flutter_sdk/MnemonicStorageHelper.kt

```diff
@@ -24,17 +24,17 @@ class MnemonicStorageHelper(context: Context) {
 
     fun getSharedPreferences(): SharedPreferences {
         val masterKey: MasterKey = Builder(localContext)
-            .setKeyScheme(MasterKey.KeyScheme.AES256_GCM)
-            .build()
+                .setKeyScheme(MasterKey.KeyScheme.AES256_GCM)
```

Comment:
Just formatted the file using https://docs.flutter.dev/tools/formatting#automatically-formatting-code-in-android-studio-and-intellij.